doc/introduction.rst

```diff
@@ -106,17 +94,51 @@ If you use eduVPN this way you need to make sure all non-Python dependies are in
 
 .. code-block:: bash
 
-    $ apt install gir1.2-gtk-3.0 gir1.2-notify-0.7 libdbus-1-dev libnotify4 python3-dateutil \
-        python3-dbus python3-nacl python3-requests-oauthlib python3-gi network-manager-openvpn \
-        python3-pip git
+   $ sudo apt install gir1.2-gtk-3.0 gir1.2-notify-0.7 python3-gi python3-requests-oauthlib \
+        python3-cryptography python3-setuptools python3-nacl python3-pytest python3-wheel \
+        python3-dbus git python3-venv pkg-config libcairo2-dev libpython3-dev \
+        gobject-introspection libgirepository1.0-dev network-manager-openvpn-gnome libdbus-1-dev
+
+For fedora:
+
+.. code-block:: bash
+
+    $ sudo dnf install -y libnotify gtk3 python3-dbus python3-requests-oauthlib \
+         python3-gobject python3-pynacl python3-pytest python3-cairo-devel \
+         gobject-introspection-devel cairo-gobject-devel dbus-python-devel
+
+Development version
+--
+
+.. code-block:: bash
+    $ git clone https://github.com/eduvpn/python-eduvpn-client.git
+    $ cd python-eduvpn-client
+
+Optionally change to another branch, e.g.
+
+.. code-block:: bash
+    $ git checkout 2.0.x
+
+Install the dependencies.
+
+For Debian or Ubuntu:
+
+.. code-block:: bash
+    $ sudo make debdev
```

Comment:
dit is dan toch dubbel? maybe better to just say `make debdev` en dan houden we daar de dependencies bij, raakt het ook niet out of date.

Reply:
Dat heb ik veel liever, ga ik aanpassen, thx.

Reply:
Done